src/main/java/com/connexta/search/common/Index.java

```diff
@@ -20,15 +20,19 @@
 import java.util.Date;
 import lombok.Builder;
 import lombok.EqualsAndHashCode;
+import lombok.Generated;
 import lombok.Getter;
+import lombok.Setter;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.solr.core.mapping.Indexed;
 import org.springframework.data.solr.core.mapping.SolrDocument;
 
 @SolrDocument(collection = SOLR_COLLECTION)
 @Getter
+@Setter
 @EqualsAndHashCode
 @Builder(toBuilder = true)
+@Generated
```

Comment:
A note to other reviewers, this annotation flags the class as generated and Jacoco will exclude it from it's coverage analysis.

Reply:
Yeah, I checked the other annotations and they don't include `@Generated`

Reply:
Oh, I didn't see the "lombok doesn't actually add this annotation" note 😆